transmart-core-db/grails-app/domain/org/transmartproject/db/i2b2data/PatientDimension.groovy

```diff
@@ -35,6 +35,7 @@ class PatientDimension implements Patient {
     // private
     String sourcesystemCd
     String sexCd
+    String inTrialId
```

Comment:
You are right, fieldConstraint is not working for this: _org.hibernate.QueryException: could not resolve property: inTrialId of: org.transmartproject.db.i2b2data.PatientDimension_